src/views/home/components/overview/overview.styles.js

```diff
@@ -28,10 +28,7 @@ const useOverviewStyles = createUseStyles(theme => ({
     display: 'inline'
   },
   batchFrequencyDecimalsOnly: {
-    display: 'none',
-    [theme.breakpoints.sm]: {
-      display: 'inline'
-    }
+    display: 'none'
```

Comment:
Why setting the css to `display: none` instead of just removing the `span` with the decimals in the HTML?

Reply:
Because they said that we are removing it "temporarily" :) But if you think is better to remove it for good, I can do that.

Reply:
Ahhh I understand! Nope, it's okay then, it was just a question 😄